### PR TITLE
Implement endpoint to update booking

### PR DESCRIPTION
**Pull request by Sviat2001**

### CODE REVIEW

**Comment by Wectro20** [comment]
|File|Coverage [72.04%]|:green_apple:|
|:-|:-:|:-:|
|[BookingNotFoundException.java](https://github.com/Wectro20/intellistart-java-2022-propositum/blob/16e37393184a837eaf477302f9d430c659e4bdaf/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fexceptions%2FBookingNotFoundException.java)|100%|:green_apple:|
|[BookingController.java](https://github.com/Wectro20/intellistart-java-2022-propositum/blob/16e37393184a837eaf477302f9d430c659e4bdaf/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fcontroller%2FBookingController.java)|100%|:green_apple:|
|[TimeSlotValidationService.java](https://github.com/Wectro20/intellistart-java-2022-propositum/blob/16e37393184a837eaf477302f9d430c659e4bdaf/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fservice%2FTimeSlotValidationService.java)|90.34%|:green_apple:|
|[BookingService.java](https://github.com/Wectro20/intellistart-java-2022-propositum/blob/16e37393184a837eaf477302f9d430c659e4bdaf/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fservice%2FBookingService.java)|81.06%|:green_apple:|
|[ApplicationExceptionHandler.java](https://github.com/Wectro20/intellistart-java-2022-propositum/blob/16e37393184a837eaf477302f9d430c659e4bdaf/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fexceptions%2FApplicationExceptionHandler.java)|31.39%|:green_apple:|

|Total Project Coverage|62.25%|:green_apple:|
|:-|:-:|:-:|

File: src/main/java/com/intellias/intellistart/interviewplanning/service/BookingService.java [hunk 1]
```diff
@@ -155,4 +156,158 @@ private List<Booking> allInterviewerBookingsForCurrentWeek(
                     .getCurrentWeekNumber().getWeekNum())).collect(Collectors.toList());
   }
 
+  /**
+   * Create booking for interview.
```
Comment (VovaMitsura) [inline src/main/java/com/intellias/intellistart/interviewplanning/service/BookingService.java:160]: Update *

**Comment by Wectro20** [comment]
|File|Coverage [72.04%]|:green_apple:|
|:-|:-:|:-:|
|[BookingNotFoundException.java](https://github.com/Wectro20/intellistart-java-2022-propositum/blob/16e37393184a837eaf477302f9d430c659e4bdaf/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fexceptions%2FBookingNotFoundException.java)|100%|:green_apple:|
|[BookingController.java](https://github.com/Wectro20/intellistart-java-2022-propositum/blob/16e37393184a837eaf477302f9d430c659e4bdaf/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fcontroller%2FBookingController.java)|100%|:green_apple:|
|[TimeSlotValidationService.java](https://github.com/Wectro20/intellistart-java-2022-propositum/blob/16e37393184a837eaf477302f9d430c659e4bdaf/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fservice%2FTimeSlotValidationService.java)|90.34%|:green_apple:|
|[BookingService.java](https://github.com/Wectro20/intellistart-java-2022-propositum/blob/16e37393184a837eaf477302f9d430c659e4bdaf/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fservice%2FBookingService.java)|81.06%|:green_apple:|
|[ApplicationExceptionHandler.java](https://github.com/Wectro20/intellistart-java-2022-propositum/blob/16e37393184a837eaf477302f9d430c659e4bdaf/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fexceptions%2FApplicationExceptionHandler.java)|31.39%|:green_apple:|

|Total Project Coverage|62.25%|:green_apple:|
|:-|:-:|:-:|

**Comment by Wectro20** [comment]
|File|Coverage [72.04%]|:green_apple:|
|:-|:-:|:-:|
|[BookingNotFoundException.java](https://github.com/Wectro20/intellistart-java-2022-propositum/blob/d1717a0ecf7001434c336c9962b1367a73d2d601/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fexceptions%2FBookingNotFoundException.java)|100%|:green_apple:|
|[BookingController.java](https://github.com/Wectro20/intellistart-java-2022-propositum/blob/d1717a0ecf7001434c336c9962b1367a73d2d601/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fcontroller%2FBookingController.java)|100%|:green_apple:|
|[TimeSlotValidationService.java](https://github.com/Wectro20/intellistart-java-2022-propositum/blob/d1717a0ecf7001434c336c9962b1367a73d2d601/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fservice%2FTimeSlotValidationService.java)|90.34%|:green_apple:|
|[BookingService.java](https://github.com/Wectro20/intellistart-java-2022-propositum/blob/d1717a0ecf7001434c336c9962b1367a73d2d601/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fservice%2FBookingService.java)|81.06%|:green_apple:|
|[ApplicationExceptionHandler.java](https://github.com/Wectro20/intellistart-java-2022-propositum/blob/d1717a0ecf7001434c336c9962b1367a73d2d601/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fexceptions%2FApplicationExceptionHandler.java)|31.39%|:green_apple:|

|Total Project Coverage|62.25%|:green_apple:|
|:-|:-:|:-:|

File: src/main/java/com/intellias/intellistart/interviewplanning/service/BookingService.java (hunk 1 above)
Comment (Sviat2001) [reply src/main/java/com/intellias/intellistart/interviewplanning/service/BookingService.java]: just changed it)

**Review by VovaMitsura — APPROVED** [review]

**Pull request merged**